Makefile

```diff
@@ -54,14 +54,21 @@ test-unit:
 
 test-integration:
 	# Runs all tests including integration and property tests
+	# We don't currently have any integration tests, but we're leaving this so we can add more.
 	go test -race -tags="integration property_test" $$(go list ./... | grep -v /vendor/ | grep -v /cmd/)
 
 testlocal: generate test
 
+swarmtest:
+	go install github.com/scootdev/scoot/binaries/setup-cloud-scoot
+	${GOPATH}/bin/setup-cloud-scoot --strategy local.local run ${GOPATH}/bin/scootapi run_smoke_test 10 1m
```

Comment:
Good catch. Thanks.

go run seems weird because you give it go files, not go packages.

So I figured you'd want to build the binary and then run it... WDYT? How can I do this right?